gamestonk_terminal/loggers.py

```diff
@@ -12,7 +12,7 @@
 import gamestonk_terminal.config_terminal as cfg
 
 logger = logging.getLogger(__name__)
-LOGFORMAT = "%(asctime)s|%(levelname)s|%(name)s|%(funcName)s|%(lineno)s|%(message)s"
+LOGFORMAT = "%(asctime)s|%(levelname)s|%(name)s|%(funcName)s|%(message)s"
```

Comment:
I'd leave lineno in. It will not make sense for start and end logging, but for the rest of log statements it can be useful.